### ISSUE TITLE
In MRF: Have additional boxes for translated PLS and translated abstract

### ISSUE THREAD

**Issue by alicia-cassidy** [dfo-osdt/osp#568]
Is it possible to add AI translations boxes for Abstract and PLS. With a similar disclaimer that these are drafts and should be read over by someone who is fluent...

**Comment by vincentauger** [comment]
Two parts: 
1. Yes, we can add additional text fields to provide this information in the MRF, but we had decided against it as we established that it would add additional work not necessarily required.
2. No problem on having AI translation feature if we decide to include bilingual text fields.

**Comment by alicia-cassidy** [comment]
Right, I remembered talking about it but didn't remember our conclusions. Maybe we can keep this as low priority for now.

 The most useful place for this might be when it comes to making the Tab module, because the Minister requires the information in the tab to be in both languages

**Comment by vincentauger** [comment]
Let's review this feature for a tab module?
